components/eventing-controller/config/manager/manager.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: eventing-controller
+  namespace: system
+  labels:
+    control-plane: eventing-controller
+spec:
+  selector:
+    matchLabels:
+      control-plane: eventing-controller
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: eventing-controller
+    spec:
+      containers:
+      - image: controller
+        args:
+        - --enable-leader-election
```

Comment:
We should ignore for now "leader-election" and start simple, as defined, with "replicas: 1"

Reply:
I agree, since this is not the helm chart which will be deployed later, I propose to tackle this later and get the foundation merged. What do you think? :)

Reply:
Will be removed in next commit :)